Unix/wsman/wsman.c

```diff
@@ -1616,6 +1616,8 @@ static void _ProcessEnumerateRequest(
     if (!msg->className || !msg->nameSpace)
     {
         _CD_ProcessEnumFailed( selfCD, enumContext );
+        __MI_Instance_Destruct(msg->base.options);
+        msg->base.options = NULL;
```

Comment:
Shouldn't EnuemerateInstance_Req_Release clean out the options? They are owned by that object.